src/converters/roam/index.ts

```diff
@@ -372,11 +379,9 @@ export class RoamConverter implements IConverter {
     }
 
     // Some dates in Roam do not have the correct date-formatted UID for some reason, so we'll try to fix those
-    if (node.title?.match(DATE_REGEX)) {
-      const dateUid = dateStringToUSDateUID(node.title);
-      if (dateUid) {
-        node.uid = dateUid;
-      }
+    const parsedDate = this.parseFlexibleDate(node.title);
+    if (parsedDate) {
+      node.uid = convertDateToTanaJournalUID(parsedDate);
     }
 
```

Comment:
Below you should not set that weird date to to ir.name, but rather transform it to YYYY-MM-DD for TanaIR.

Sorry about this being so bad.